VsGallery.Core/VsixFeed/VsixFeedBuilder.cs

```diff
@@ -137,7 +137,7 @@ private SyndicationFeed CreateNewFeed()
                 // through the DownloadModule for tracking, if tracking is off then serve the static files directly 
                 // from the file-system
                 item.Content = SyndicationContent.CreateUrlContent(_configGallery.TrackDownloads 
-                                                                   ? new Uri($"/api/download/{pkg.Id}/{pkgFileName}", UriKind.Relative) 
+                                                                   ? new Uri($"../api/download/{pkg.Id}/{pkgFileName}", UriKind.Relative) 
```

Comment:
Hi David, thanks for this PR. I'm concerned that this may not be the best solution to this problem you raised. Perhaps a more stable solution would be to prepend a "baseurl" to the path that takes into account the root of the microservice?

Reply:
t's going well. But it didn't apply to the master sauce.